WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsViewModel.swift

```diff
@@ -98,6 +110,13 @@ class SiteStatsInsightsViewModel: Observable {
         ActionDispatcher.dispatch(InsightAction.refreshInsights)
     }
 
+    private func fetchStatsForInsightsLatestPost(postID: Int) {
```

Comment:
Hey @stevebaranski . 

1. It doesn't appear the chart is updated when Insights in refreshed. I think just adding this to `refreshInsights()` will do it:
```
        if let postID = insightsStore.getLastPostInsight()?.postID {
            ActionDispatcher.dispatch(PeriodAction.refreshPostStats(postID: postID))
        }
```
2. This is so nitpicky, but if you move `fetchStatsForInsightsLatestPost` into the private extension, you won't need to mark it as `private`.

Reply:
Thanks for the feedback, @ScoutHarris. I have applied the changes you requested via an additional commit. I believe this is ready for another look.